### ISSUE TITLE
AS 1/2 New Method

### ISSUE THREAD

**Issue by GoogleCodeExporter** [google-code-export/asdec#39]
```
What steps will reproduce the problem?
1. Try to decompile SWF file which contains method "new" invocations.
I took an example here: http://www.samsungdforum.com/Guide/tut00084/index.html
You can find the original source there too.

What is the expected output? What do you see instead?
You get it in P-code but it isn't translated into source code.

function getMute(isMute)
{
   if(isMute)
   {
   }
}

vs

function getMute(isMute:Number)
{
    if (isMute) {
        new Tween(mute_mc,"_alpha",Strong.easeOut,0,100,0.1,true);
    }
    else {
        new Tween(mute_mc,"_alpha",Strong.easeOut,100,0,0.1,true);
    }
}

What version of the product are you using? On what operating system?
1.3.0 - 1.4.0u1
Windows 7 64 bit

Could you provide the SWF file you have problem with?
If the answer is yes, then please attach it here or send me it via email.
Could you at least attach PCode source?

Do you have the original source code which produced the wrong
decompilation? If yes, then please attach it.

Please provide any additional information below.


```

Original issue reported on code.google.com by `plisnich...@gmail.com` on 5 Mar 2013 at 9:37

Attachments:
- [external_call.swf](https://storage.googleapis.com/google-code-attachments/asdec/issue-39/comment-0/external_call.swf)

**Comment by GoogleCodeExporter** [comment]
Original comment by `jindra.p...@gmail.com` on 11 Mar 2013 at 6:06
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
Original comment by `jindra.p...@gmail.com` on 11 Mar 2013 at 6:07

**Comment by GoogleCodeExporter** [comment]
```
Hi,
it is now fixed in version 1.4.2, try it.
```

Original comment by `jindra.p...@gmail.com` on 24 Mar 2013 at 11:49

**Comment by GoogleCodeExporter** [comment]
Original comment by `jindra.p...@gmail.com` on 24 Mar 2013 at 11:50
- Changed state: **Fixed**
